docs/elements/guides/sign-in.mdx

```diff
@@ -0,0 +1,267 @@
+
+        <Clerk.Field name="identifier">
+          <Clerk.Label>Email</Clerk.Label>
+          <Clerk.Input />
+          <Clerk.FieldError />
+        </Clerk.Field>
+
+        <SignIn.Action submit>Continue</SignIn.Action>
+      </SignIn.Step>
+    </SignIn.Root>
+  )
+}
+```
+
+`<Clerk.Field>` takes care of wiring up the input with the label element, and `<Clerk.FieldError>` will render any field-specific errors that get returned from Clerk's API. You've also used the `<SignIn.Action>` component. This component provides common actions that are used throughout the flows. In this case, you're using the `submit` action to render a submit button for the start form.
+
+<Callout type="info">
+If your Clerk instance supports signing in with Google and doesn't require MFA, you should be able to complete a sign-in with the components you've rendered so far!
+</Callout>
+
+### Add verification
+
+As one progresses through a sign-in attempt, your users may be asked to **verify** a number of authentication factors. This is done in the `verifications` step. Depending on your instance configuration and the state of the sign-in attempt, certain strategies will require different fields. To support conditional rendering based on the strategy being verified, you will use the `<SignIn.Strategy>` component. Assuming that your instance is configured to accept email codes, add the code necessary to accept them:
+
+```tsx filename="/app/sign-in/[[...sign-in]]/page.tsx" {25-38}
+'use client'
+
+import * as Clerk from '@clerk/elements/common'
+import * as SignIn from '@clerk/elements/sign-in'
+
+export default function SignInPage() {
+  return (
+    <SignIn.Root>
+      <SignIn.Step name="start">
+        <h1>Sign in to your account</h1>
+
+        <Clerk.Connection name="google">
+          Sign in with Google
+        </Clerk.Connection>
+
+        <Clerk.Field name="identifier">
+          <Clerk.Label>Email</Clerk.Label>
+          <Clerk.Input />
+          <Clerk.FieldError />
+        </Clerk.Field>
+
+        <SignIn.Action submit>Continue</SignIn.Action>
+      </SignIn.Step>
+
+      <SignIn.Step name="verifications">
+        <SignIn.Strategy name="email_code">
+          <h1>Check your email</h1>
+          <p>We've sent a code to <SignIn.SafeIdentifier />.</p>
+
+          <Clerk.Field name="code">
+            <Clerk.Label>Email code</Clerk.Label>
+            <Clerk.Input />
+            <Clerk.FieldError />
+          </Clerk.Field>
+
+          <SignIn.Action submit>Continue</SignIn.Action>
+        </SignIn.Strategy>
+      </SignIn.Step>
+    </SignIn.Root>
+  )
+}
+```
+
+In the verifications step, you will need to render a `<SignIn.Strategy>` component for each authentication strategy that your instance supports. This will also handle multi-factor authentication (MFA) if it is enabled for your instance. Check out the [Sign-in reference](/docs/elements/reference/sign-in#strategy) for a full list of possible authentication strategies.
```

Comment:
```suggestion
In the verifications step, you must render a `<SignIn.Strategy>` component for each authentication strategy that your instance supports. This will also handle MFA if it is enabled for your instance. Check out the [Sign-in reference](/docs/elements/reference/sign-in#strategy) for a full list of possible authentication strategies.
```